gravitee-apim-gateway/gravitee-apim-gateway-services/gravitee-apim-gateway-services-sync/src/main/java/io/gravitee/gateway/services/sync/process/distributed/mapper/ApiMapper.java

```diff
@@ -61,7 +61,17 @@ public ApiMapper(ObjectMapper objectMapper, SubscriptionMapper subscriptionMappe
                     parseAndAssert(
                         payload,
                         io.gravitee.gateway.reactive.handlers.api.v4.Api.class,
-                        reactableApi -> reactableApi.getDefinitionVersion() == DefinitionVersion.V4
+                        reactableApi ->
+                            reactableApi.getDefinitionVersion() == DefinitionVersion.V4 &&
+                            ((io.gravitee.definition.model.v4.Api) reactableApi.getDefinition()).getType() != ApiType.NATIVE
```

Comment:
I would have tested if the type is `PROXY` or `MESSAGE` instead so it stays consistent if we add a new type. WDYT?

Reply:
you're right